### ISSUE TITLE
Time sheet or time reporting

### ISSUE THREAD

**Issue by chupzzz** [hcengineering/platform#6357]
This is not a bug report but a question.

**How can I see all the time spent on all completed tasks?** We're using _spent time_ field and it's saves times for the task. But how can I see an aggregated report in all tasks for time period? Or maybe you have different vision/approach for using this feature (time sheet maybe)?

Thanks

**Comment by asmit-gupta** [comment]
Yes, indeed, a very required feature. Also, maybe we can have a report section in this? where we can see how many tasks (issues) have been done in some time frame and remarks can be written accordingly (or does this exist, I am missing it?).
